styles/style.css

```diff
@@ -0,0 +1,237 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500;600&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-size: 30px;
+    font-family: 'Montserrat', sans-serif;
+    color: black;  
+}
+
+
+.page {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    background-image: url(../img/QuizBackground.png);
+    background-size: cover;
+    padding: 3%;
+}
+
+.menu{
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    width: 100%;
+    margin-bottom: 2%;
+}
+
+.menu__logo {
+    width: 25%;
+}
+
+.menu__list {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    width: 65%;
+    height: 110px;
+    background-color: white;
+    border-radius: 16px;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25), 4px 4px 40px rgba(255, 0, 0, 0.9);
+}
+
+.menu__item {
+    position: relative;
+    padding: 10px;
+    width: 20%;
+    text-decoration: none;
+    font-weight: 600;
+    text-align: center;
+    color: black;
+    
+}
+
+.menu__item--selected::before {
+    content: "";
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    height: 6px;
+    background: rgba(255, 0, 0, 0.5);
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25), 4px 4px 40px rgba(255, 0, 0, 0.9);
+    border-radius: 4px;
+}
+
+.menu__item--unselected {
+    color: rgba(0, 0, 0, 0.4);
+}
+
+.main {
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+}
+
+.question__photo {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    width: 25%
+}
+
+.question__image {
+    width: 100%;
+}
+
+.mode {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    width: 65%;
+}
+
+.mode__type, .mode__rules {
+    background-color: white;
+    width: 100%;
+    text-align: center;
+    padding: 20px 40px;
+    margin-bottom: 3%;
+    border-radius: 16px;
+}
+
+.mode__type {
+    font-weight: 600;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25), 4px 4px 40px rgba(255, 0, 0, 0.9), inset 4px 5px 4px rgba(0, 0, 0, 0.5);
+}
+
+.mode___description {
+    margin-top: 10px;
+    font-size: 0.8rem;
+    text-align: left;
+}
+
+.buttons {
+    width: 65%;
+    display: flex;
+    justify-content: space-between;
+    margin-left: 32%;
+}
+
+.button {
+    border-radius: 16px;
+    border: none;
+    padding: 20px 30px;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25); 
+    background-color: white;
+}
+ .button--ranking {
+    width: 48%;
+ }
+
+.button--play {
+    width: 48%;
+    background-color: rgb(255, 0, 0);
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25), 4px 4px 40px rgba(255, 0, 0, 0.9);
+    text-transform: uppercase;
+    color: white;
+}
+
+.button--settings {
+    position: absolute;
```

Comment:
czemu ten przycisk jest stylowany position: absolute? :)
nie wiem czy celowo, na mobile jest szerszy niż te przyciski powyżej bo ma width ustawiony procentowo

Reply:
w desktopie jest position absolute, ponieważ nie umiałam inaczej go ustawić w odpowiednim miejscu. Jak wrzucałam wszystkie przyciski do jednego diva i próbowałam flexem to wtedy te dwa z prawej nie zgrywały mi się z sekcją, rozjeżdżały się, więc to jedyny pomysł jaki wpadł mi do głowy.

Co do mobile, to ustawiłam póki co na 100% zamiast na 95% jak pozostałe sekcje, bo jak miałam 95% to nie potrafiłam go wycentrować :/ margin: 0 auto nie działało. Macie jakiś pomysł jak to zrobić?